### DISCUSSION TITLE
Duda para back.

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#576] [Q&A]
Hola, una duda para entrar al back, es la liga de nuestro perfil¿?, porque si es de un repo lo de front end los puse en diferentes repositorios.

Pregunto aqui porque es más tranquilo, jaja ya se descontrolo lo el teams.

**Comment by cesaralejandroramirezalcaraz** [comment]
Es correcto, es la liga de tu perfil principal de github.

**Comment by JossPerezT** [comment]
Así es, solo para que chequen que estés en la organización.

**Comment by GloriaCoralCerecedo** [comment]
Muchas gracias
